src/Components/ChangePasswordBox/ChangePasswordBox.jsx

```diff
@@ -0,0 +1,77 @@
+import axios from 'axios';
+import React, { useState } from 'react'
+import { useAuth } from '../../Context/Video-Context';
+
+function ChangePasswordBox({ currentState, setCurrentState }) {
+    
+    const { state: { currentUserId } } = useAuth();
+    const [message, setMessage] = useState("");
+
+    async function serverPostPassword() {
+        const data = await axios.post(`https://Sparrow-Media-Authentication.prakhar10v.repl.co/users/${currentUserId}`, {
+            username: currentState.username,
+            email: currentState.email,
+            password : currentState.newPassword
+        })
+        if (data) {
+            setMessage(true)
+            setCurrentState({
+                oldPassword: "",
+                newPassword: "",
+                confirmPassword : ""
+            })
+        } else {
+            setMessage(false)            
+        }
+    }
+
+    function submitHandler(e) {
+        e.preventDefault();
+        if (currentState.oldPassword === currentState.actualPassword && currentState.newPassword === currentState.confirmPassword) {
```

Comment:
Can avoid `if...else` block and use ternary operator instead

Reply:
Used `if...else` here, for checking two conditions, at once. Using Ternary Operator might affect Code Reusability.